src/utils/src/arg_parser.rs

```diff
@@ -277,6 +278,16 @@ impl<'a> Arguments<'a> {
             return Ok(());
         }
 
+        // If `--version` is provided as a parameter, we artificially skip the parsing of other
+        // command line arguments by adding just the version argument to the parsed list and
+        // returning.
+        if args.contains(&VERSION_ARG.to_string()) {
+            let mut version_arg = Argument::new("version").help("Print the binary version number.");
+            version_arg.user_value = Some(Value::Bool(true));
+            self.insert_arg(version_arg);
+            return Ok(());
+        }
```

Comment:
We should be able to add a test case for this code path. Isn't there one for parsing help as well?

Reply:
Yep, sorry, it is indeed.
@Pennyzct can you, please, update `test_parse()` with a test also for `--version` argument?